what-is-chalk.mdx

```diff
@@ -18,215 +18,436 @@ import { PyDiffEditor } from '@/components/Editor'
 
 <DocsHero />
 
-Chalk is a feature store that enables innovative machine learning teams
-to focus on building the unique products and models that make their
-business stand out. Behind the scenes, Chalk seamlessly handles
-data infrastructure with a best-in-class developer experience.
-Here’s how it works.
+Chalk is a real-time data platform that enables innovative teams to build, evaluate, and deploy enterprise AI and ML systems.
 
----
-
-## Develop
+Chalk simplifies
 
-Chalk makes it simple to develop feature pipelines for machine
-learning. Define Python functions using the libraries and tools you're
-familiar with instead of specialized DSLs. Chalk then orchestrates
-your functions into pipelines that execute in parallel on a Rust-based
-engine and coordinates the infrastructure required to compute
-features.
+- deploying production-grade infrastructure for low-latency (< 10ms) inference at scale
+- developing feature pipelines - use pure Python to express our features as data classes and write functions with libraries and tools we're familiar with instead of specialized DSLs
+- experimenting and iterating on models - data-as-code enables model versioning and A/B
+- sharing features and collaborating across teams with branch deploys (single source of truth with data-as-code)
+- building datasets for training and simulating models with historical production traffic
+- incorporating unstructured data sources with LLMs into ML pipelines
 
-### Features
+## How do I express my features in Chalk?
 
-To get started, [define your features](/docs/features) with
-[Pydantic](https://pydantic-docs.helpmanual.io/)-inspired Python classes.
-You can define schemas, specify relationships, and add metadata
-to help your team share and re-use work.
+To get started, we define our features with Pydantic-inspired Python classes.
+We can define schemas, specify relationships, materialize/tile features, assign ownership, and isolate features across environments, all with code.
 
 ```py
-from chalk import _, DataFrame
 import chalk.functions as F
-from chalk.features import features
+from chalk import features, DataFrame, Windowed, windowed, _
 
 @features
 class Transaction:
     id: int
     amount: float
-    user_id: "User.id"
-    user: "User"
+    discount_percentage: float
+    is_expensive_purchase: bool = _.amount > 100 # create new features inline with Chalk Expressions
```

Comment:
let's lift these comments next to the code to above -- it makes these lines pretty long